client/core/core.go

```diff
@@ -1028,66 +1028,75 @@ func (c *Core) encryptionKey(pw []byte) (encrypt.Crypter, error) {
 	return crypter, nil
 }
 
+func (c *Core) storeDepositAddress(wdbID []byte, addr string) error {
+	// Store the new address in the DB.
+	dbWallet, err := c.db.Wallet(wdbID)
+	if err != nil {
+		return fmt.Errorf("error retreiving DB wallet: %w", err)
+	}
+	dbWallet.Address = addr
+	return c.db.UpdateWallet(dbWallet)
+}
```

Comment:
I had actually started on that but I realized that the address is part of the full wallet encoding (e.g. `wBkt.Put(walletKey, wallet.Encode())`), so it would be a full load/modify/store like this anyway, unlike with balance that is in a different bucket allowing a clean `(*BoltDB).UpdateBalance`.  I suppose it would be somewhat more efficient and in a single db transaction though.

Reply:
I suppose the "right" solution would be to remove the address from the encoding and give it it's own key, but that would require a DB update, and I'm fine with load -> modify -> store for now.

Reply:
Yeah, I didn't want to go down the DB upgrade path because this is 0.1.4 material, but that would make sense for 0.2 and beyond.
To clarify, you'd still want a BoltDB method or just run with this for now given the encoding issue?

Reply:
We can follow up.